{"uid":"84d628750ed23106","name":"displays two todo items by default","fullName":"example to-do app.displays two todo items by default","historyId":"46a278017f3323391c62ae8c2fbc99aa","time":{"start":1666186436450,"stop":1666186436631,"duration":181},"description":"","descriptionHtml":"","status":"broken","statusMessage":"`cy.visit()` must be called with a `url` or an `options` object containing a `url` as its 1st argument\n                \n                https://on.cypress.io/visit\n                \n                Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `example to-do app`","statusTrace":"CypressError: `cy.visit()` must be called with a `url` or an `options` object containing a `url` as its 1st argument\n                \n                https://on.cypress.io/visit\n                \n                Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `example to-do app`\n                    at Context.visit (http://localhost:43783/__cypress/runner/cypress_runner.js:141774:78)\n                    at wrapped (http://localhost:43783/__cypress/runner/cypress_runner.js:157372:43)\n                From Your Spec Code:\n                    at Context.eval (webpack:///./cypress/e2e/tests/todo.cy.js:21:11)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"suite","value":"example to-do app"},{"name":"testClass","value":"example to-do app"},{"name":"testMethod","value":"displays two todo items by default"},{"name":"package","value":"example to-do app"},{"name":"resultFormat","value":"allure1"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"84d628750ed23106.json","parameterValues":[]}